src/packages/@workspaces/features/rest-explorer/components/http-url-section/HttpUrlSection.svelte

```diff
@@ -179,13 +181,22 @@
       onUpdateRequestState({ requestSplitterDirection: e.detail });
     }}
   /> -->
-  <Tooltip title={"Save"} placement={"bottom"} distance={12} zIndex={10}>
+  <Tooltip
+    title={isGuestUser ? "Login to save" : "Save"}
+    show={requestUrl !== ""}
+    placement={"bottom-left"}
+    distance={12}
+    zIndex={10}
+  >
     <button
       class="ms-2 save-disk d-flex align-items-center justify-content-center border-radius-2 border-0"
       on:click={handleSaveRequest}
       on:mouseenter={handleMouseEnter}
       on:mouseleave={handleMouseLeave}
-      disabled={isSave || userRole === WorkspaceRole.WORKSPACE_VIEWER
+      disabled={isSave ||
+      requestUrl === "" ||
+      userRole === WorkspaceRole.WORKSPACE_VIEWER ||
+      isGuestUser
         ? true
         : false}
          style="background-color: {isSave ? 'var(--icon-secondary-550)' : 'var(--bg-secondary-400)'}; color: white;"
```

Comment:
api save button cant be dependent on request url or guest user 
cc: @nagesh953 @Monika-APM  @astitva-techdome